### ISSUE TITLE
U7_snRNA

### ISSUE THREAD

**Issue by sjm41** [The-Sequence-Ontology/SO-Ontologies#508]
**What is this request referring to?**
U7 small nuclear RNA

**What is the name you would like SO to give the term?**
U7_snRNA

**What is the definition that you would like for this term?**
A 57–71 nucleotide RNA that is a component of the U7 small nuclear ribonucleoprotein complex (U7 snRNP). The U7 snRNP is required for histone pre-mRNA processing. 

**Parent Term**
snRNA (SO:0000274)

**Relevant Publications**
Schümperli D, Pillai RS. The special Sm core structure of the U7 snRNP: far-reaching significance of a small nuclear ribonucleoprotein. Cell Mol Life Sci. 2004 Oct;61(19-20):2560-70. doi: 10.1007/s00018-004-4190-0. PMID: 15526162.

https://rfam.org/family/RF00066

**Comment by davidwsant** [comment]
Hi @sjm41 ,

Thank you for bringing this to our attention. The following term has been added to SO:

id: SO:0002338
name: U7_snRNA
def: "A 57–71 nucleotide RNA that is   a component of the U7 small nuclear ribonucleoprotein complex (U7 snRNP). The   U7 snRNP is required for histone pre-mRNA processing." [PMID:15526162]
synonym: "U7 snRNA" EXACT []
synonym: "snRNA U7" EXACT []
synonym: "small nuclear RNA U7"   EXACT []
synonym: "U7 small nuclear RNA"   EXACT []
is_a: SO:0000274 ! snRNA

The [SO Browser](http://sequenceontology.org/browser/obob.cgi) should update with this change within the next 24 hours.

Best,

Dave

**Comment by sjm41** [comment]
Many thanks!

**Comment by sjm41** [comment]
Hi @davidwsant 
The definition for this in the SO browser begins with "A 57b71 nucleotide" - looks like there's a 'b' instead of a hyphen?

**Comment by davidwsant** [comment]
Hi Steven,

I see the 'b' in the browser as well. However, this is not a typo in the files. I have checked the obo and owl files and both have a dash. I think it might be a problem in the rendering in JavaScript. I have changed the dash to a ' to '. I will push this change to GitHub and see if it fixes this. If not, then I will ask someone to clear the cache for the website and hopefully that can fix it. 

Best,

Dave
